src/mon/MgrMap.h

```diff
@@ -209,6 +235,18 @@ class MgrMap
       }
       DECODE_FINISH(p);
     }
+    void dump(ceph::Formatter *f) const {
```

Comment:
nit: let's be consistent with this `{`.

Reply:
Done, also fixed all other dump function that already had inconsistent use of {